src/constants.js

```diff
@@ -4,5 +4,6 @@ export default {
   SIZES: ['s', 'm', 'l', 'xl'],
   PLACEMENTS: ['left', 'center', 'right'],
   SCALES: ['big', 'small'],
-  ICON_SIZES: ['tiny', 'small', 'medium', 'large']
+  ICON_SIZES: ['tiny', 'small', 'medium', 'large'],
+  MOBILE_NAV_ID: 'mobile-nav'
```

Comment:
If this constant is not going to be used anywhere except `Navbar.js` maybe we could keep it there